### DISCUSSION TITLE
MultiSelect with Input - Index Error (refresh)

### DISCUSSION THREAD

**Discussion by marinkreso95** [zauberzeug/nicegui#3950] [Q&A]
### Question

I use multi-select and I have an issue with editing selections after a refresh . The behaviour is the following:
- User can add trigger an action that refreshed the screen (through refreshable). When that happens I want to remember currently selected values
- I do that by storing selection, and then when page refreshes I would check if I have something stored and use tournament_selection.set_value(`tournaments_list_from_memory`)

Everything works well besides one thing. After I refresh the screen, I'm unable to edit the selection. Can't add new values nor can I delete memorized item by clicking on a x next to option (I'm using chips). Maybe user is not satisfied with the select and wants to edit it further. I get the following error:

```
Traceback (most recent call last):
  File "/Users/user/.pyenv/versions/3.10.12/envs/nicegui/lib/python3.10/site-packages/nicegui/events.py", line 400, in handle_event
    result = handler(arguments) if expects_arguments else handler()
  File "/Users/user/.pyenv/versions/3.10.12/envs/nicegui/lib/python3.10/site-packages/nicegui/elements/mixins/value_element.py", line 31, in handle_change
    self.set_value(self._event_args_to_value(e))
  File "/Users/user/.pyenv/versions/3.10.12/envs/nicegui/lib/python3.10/site-packages/nicegui/elements/select.py", line 87, in _event_args_to_value
    args = [self._values[arg['value']] if isinstance(arg, dict) else arg for arg in e.args]
  File "/Users/user/.pyenv/versions/3.10.12/envs/nicegui/lib/python3.10/site-packages/nicegui/elements/select.py", line 87, in <listcomp>
    args = [self._values[arg['value']] if isinstance(arg, dict) else arg for arg in e.args]
IndexError: list index out of range
```
This is my ui.select
```py
tournament_selection = ui.select(
    label='Tournaments',
    multiple=True,
    on_change=lambda i=i, dialog=dialog: on_tournament_change(i, dialog),
    with_input=True,
    options=sorted(list(set([x['TOURNAMENT'] for x in all_data]))),
).classes('w-60').props('use-chips').props('outlined')
 ```

**Comment by falkoschindler** [comment]
Hi @marinkreso95,

Can you, please, try to create a minimum reproducible code example, including the refreshable you mentioned? This would allow us and the community to help more efficiently. Thanks!
[Why is it important to provide a minimal reproducible example?](https://github.com/zauberzeug/nicegui/wiki/FAQs#why-is-it-important-to-provide-a-minimal-reproducible-example)
